Include/httpClient/httpClient.h

```diff
@@ -496,6 +527,38 @@ STDAPI HCHttpCallRequestSetSSLValidation(
 ) noexcept;
 #endif
 
+/////////////////////////////////////////////////////////////////////////////////////////
+// HttpCallResponse Set APIs
+//
+
+/// <summary>
+/// The callback definition used by an HTTP call to write the response body.
+/// </summary>
+/// <param name="call">The handle of the HTTP call.</param>
+/// <param name="source">The source from which bytes may be read.</param>
+/// <param name="bytesAvailable">The maximum number of bytes that can be read from the source.</param>
+/// <param name="bytesRead>The number of bytes that were actually written to the source.</param>
+/// <returns>Result code for this callback. Possible values are S_OK, E_INVALIDARG, or E_FAIL.</returns>
+typedef HRESULT
+(CALLBACK* HCHttpCallResponseBodyWriteFunction)(
```

Comment:
should probably be marked noexcept

Reply:
I tried that initially but it turns out `noexcept` is not allowed in a typedef declaration. I think might be why the other callback types (HCMemAllocFunction, HCMemFreeFunction,  HCWebSocketMessageFunction, ...) don't have the `noexcept` qualifier either